### ISSUE TITLE
Accessibility: No headings on page

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#430]
**Description**
Certain exhibitions do not have any headings on a page. All pages should have headings, beginning with only a single H1 and then going from there. This issue occurs when there are no headings on a page at all.

**Solution**
Define a heading structure for all pages. The H1 for all pages would be the title of the Exhibit and then allow users to begin using headings beginning with H2 within their content boxes. This would require content owners who are currently using an H1 within their exhibit pages to change them. This is the method currently used on all WordPress unit websites, except we define both H1 and H2 in a template, with content owners starting with H3.
